docs/started/releasepolicy.md

```diff
@@ -81,3 +81,53 @@ server.
 | (0.3)           |                                           | 2015-03        | 2015-07              |
 | (0.2)           |                                           | 2014-10        | 2015-03              |
 | (0.1.7)         |                                           | 2014-01        | 2014-10              |
+
+## Secondary Rucio Software Policy
+
+### WebUI
+
+The Rucio WebUI is an initiative to modernize the Rucio user interface by
+leveraging the latest web technologies. Built with Next.js, TailwindCSS, and
+React.js, this new WebUI aims to provide a robust architecture designed to
+serve the community for the next decade, much like its predecessor.
+
+#### Key Technologies
```

Comment:
Remove, no need to mention this here.

Also the text above, if we can just link to a rucio-webui page in the doc, we don't need to mention the entire text.